pkg/cmd/grafana-server/server.go

```diff
@@ -231,27 +243,28 @@ func (g *GrafanaServerImpl) writePIDFile() {
 	g.log.Info("Writing PID file", "path", *pidFile, "pid", pid)
 }
 
-func sendSystemdNotification(state string) error {
+func (g *GrafanaServerImpl) sendSystemdNotification(state string) {
```

Comment:
@marefr I decided to make the change that we don't fail if unable to write to systemd, only log the failure. Please advise on whether this makes sense or not. I figured that it's not a hard error if we can't write to systemd, but I could be wrong.